### PR TITLE
6pm 1 fix about page

### LINKED ISSUE #100: Change the "About" page to include a description of the CS LAs app
# User Story

As a user I can go to the About page to learn more on what the CS LAs is and what it does.

# Acceptance Criteria

- [x] About page should have an accurate description of the CS LAs app
- [x] About page has a Purpose heading describing the purpose of CS LAs app
- [x] About page has a Features heading describing the features of CS LAs app
- [x] About page has a section mentioning CS156's role in creating/maintaining CS LAs app along with a link to the GitHub

# Implementation Todos

Front end:

- [x] Left align headings
- [x] Main Heading should be About Us
- [x] Subheadings: Purpose, Features, Team
- [x] Short description under each subheading

Back end:

None

Testing:

- [x]  All previous tests still pass

Guidance/Information:
- You have Creative Freedom to design the About page along with any content in it.  You might look at what Prof. Conrad wrote in the #proj-ucsb-cs-las channel on the Slack about the overall purpose of the app, and/or ask the TAs/LAs that are familiar with the app for guidance.
- Changes should be in [`javascript/src/main/pages/About/About.js`](https://github.com/ucsb-cs156-w21/proj-ucsb-cs-las/blob/main/javascript/src/main/pages/About/About.js)
- More on Typography in [Bootstrap](https://getbootstrap.com/docs/4.0/content/typography/)
- Headings are usually wrapped in `<h1>` tags with subheadings `<h2>`, `<h3>` and so on.
- Content after headings are wrapped in `<p>` tags
- About page already has an example of this heading then paragraph layout, we just need to change it and add more headings!

### PR DESCRIPTION
**Pull request by justinsurmani**
Changed the About Page to include a description of the CS LAs app as well as a link to the repo on Github.

Closes #100

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,3 +1,4 @@
+app.member.hosted-domain=ucsb.edu
```
Comment (LeoJr99) [inline src/main/resources/application.properties:1]: Not sure if this is supposed to be passed alongside the rest of the code.

**Review by LeoJr99 — COMMENTED** [review]
Everything looks good and meets the issue requirements. 
However extra line was passed in for application.properties.

File: src/main/resources/application.properties (hunk 1 above)
Comment (justinsurmani) [reply src/main/resources/application.properties:1]: Correct, this line was not meant to be pushed

**Review by LeoJr99 — APPROVED** [review]
LGTM

**Review by pconrad — CHANGES_REQUESTED** [review]
Please do NOT delete the file data.sql.SAMPLE.
That is an important file for other teams.

**Review by pconrad — APPROVED** [review]
LGTM

**Review by LeoJr99 — APPROVED** [review]
LGTM

**Pull request merged**